ivy/functional/frontends/numpy/mathematical_functions/rounding.py

```diff
@@ -32,6 +32,19 @@ def _ceil(
         ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
     return ret
 
+#round
+@handle_numpy_out
+@handle_numpy_dtype
+@to_ivy_arrays_and_back
+@handle_numpy_casting
+@from_zero_dim_arrays_to_scalar
+def round(
+    x,
+    decimals=0,
+    out=None,
+):
+    return ivy.round(x, decimals=decimals, out=out)
```

Comment:
ivy.round does not have an argument `decimals` which you are passing in here.

Reply:
My bad it was a mistake. 'around' function has decimals but not 'round' function.